### PR TITLE
Vehicles spawn logic

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

Changed logic for vehicles spawn on enemy bases
removed helis from spawn pool on runways
removed gunships from hangars spawn pool
added IFVs and airborne vehicles as heavy patrol vehicles
added light apc to spawn pool
added more vehicle types to patrol pool

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf [hunk 1]
```diff
@@ -246,7 +246,8 @@ if (_spawnParameter isEqualType []) then {
 		} else {
 			(_faction get "vehiclesMilitiaTrucks") +
 			(_faction get "vehiclesMilitiaLightArmed") +
-			(_faction get "vehiclesMilitiaCars")
+			(_faction get "vehiclesMilitiaCars")+
+			(_faction get "vehiclesBasic") //we should use them somewhere at least
```
Comment (Silence-112) [inline A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: basic shouldn't be used anywhere outside of where it already is used, for reasons i've already mentioned

Comment (wersal454) [reply A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: That's the point, they aren't used anywhere

Comment (Silence-112) [reply A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: That's my point, there's a reason they aren't used anywhere

I don't see how people would like going to an outpost just to find a quadbike, or another vehicle of questionable quality

Comment (wersal454) [reply A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: And here is another point, you aren't getting vehicles that you want all the time on mil bases or air bases, right? Sometimes you get shitty vehicles. + I believe basics will spawn only on low war levels alongside shitty militia vehicles.

Comment (Silence-112) [reply A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: ~~I don't see anything in that function to only spawn basics at low war levels~~

Everything else looks fine, I just don't see any reason to start using basic now

Comment (Silence-112) [reply A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf:250]: Nevermind, forgot the barbolani spaghetti when it was still hardcoded to "FIA" for low war level

**Review by Silence-112 — APPROVED** [review]
I suppose we can see how `vehiclesBasic` works out

**Pull request merged**